### DISCUSSION TITLE
Search result Multi Problems

### DISCUSSION THREAD

**Discussion by DraculaVladimir** [immich-app/immich#11219] [Feature Request]
### The bug

Immich does not allow to easily select multiple images when the search results appear using SHIFT+LEFT_CLICK

Search results appear out of order. I Believe that the results should showup relative in chronological order

It would also be interesting to apply multiple filters at once. For example showing all images that where taken on a these set of cameras insterad of opnly being able to select ONE model or camera make at ime.

### The OS that Immich Server is running on

Unraid

### Version of Immich Server

v1.109.1

### Version of Immich Mobile App

NA

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
1. Search for dog
2. Images of dogs appear
3. Images are out of chronological order
4. Images cannot be selected by using SHIFT+LEFT_CLICK

...
```

**Comment by alextran1502** [comment]
I am converting this into a feature request. We are gonna circle back to this feature and add some of the enhancement you mentioned above

**Comment by bo0tzz** [comment]
> Search results appear out of order. I Believe that the results should showup relative in chronological order

Search results are ordered by relevance

**Comment by DraculaVladimir** [comment]
I think users would like to have the option to choose how the data is
displayed. But thank you for clarifying that it is in order.
